README.md

```diff
@@ -1 +1,25 @@
-# Responsive-Design
\ No newline at end of file
+# Responsive-Design
+BUILDING WITH RESPONSIVE DESIGN
+
+Microverse project
+
+This project consists of building an HTML document that matches the appearance of https://thenextweb.com/ web page.
+
+You can find the original project specification at:
+ <https://www.theodinproject.com/courses/html5-and-css3/lessons>  
+<https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-responsive-design>
+
+Author:
+This project was executed by Mark Baidebura and Kyankya Raymond students of Microverse.
```

Comment:
**[REQUIRED]**
Great job creating a descriptive readme file. But it's also a best practice to add some styles to it. By doing that you are making it more easy to read and understand. And it also gives a professional look. Check out this [template](https://github.com/othneildrew/Best-README-Template) about creating awesome readme files.